src/js/components/CreateServiceModalServicePickerOption.js

```diff
@@ -0,0 +1,48 @@
+import {Hooks} from 'PluginSDK';
+import React from 'react';
+
+import CreateServiceModalServicePickerOptionContent from './CreateServiceModalServicePickerOptionContent';
+import CreateServiceModalServicePickerOptionImage from './CreateServiceModalServicePickerOptionImage';
+import Panel from './Panel';
+
+function getChildOfType(children, type) {
+  return React.Children.toArray(children).filter(
+    (child) => child.type === type
+  );
+}
+
+function CreateServiceModalServicePickerOption({children, onOptionSelect}) {
+  const columnClass = Hooks.applyFilter(
```

Comment:
^^ following up here: https://github.com/dcos/dcos-ui/pull/1782#discussion_r96471438, I'll create some logic to do this grid calculation dynamically.